CHANGES.md

```diff
@@ -1,3 +1,10 @@
+# lang_tester 0.3.12 (2020-07-13)
+
+* Failed stderr/stdout tests now use fm to show the offending line and upto 3
```

Comment:
Typo. "upto" -> "up to".

Reply:
Fixed in https://github.com/softdevteam/lang_tester/pull/69/commits/9b04ef3d6a854173ac0b7f14268e3958b600f97d.